portal/src/it/resources/application.conf

```
@@ -16,6 +16,13 @@ auth {
   }
 }
 
+misc {
+    filtersDuration {
+        timeToLive = 2 seconds
+        cleanInterval = 1 seconds
+    }
```

When you are giving a name to something, try to give information about what it's doing in terms of high-level language. It's obvious that you are talking about durations here (because of `seconds`), so the name is not very useful IMO. It's not so obvious what filters will do with these durations.
```suggestion
    filtersCleanup {
        timeToLive = 2 seconds
        interval = 1 seconds
    }
```

---

fixed